Cargo.toml

```diff
@@ -14,7 +14,7 @@ exclude = [
 
 [dependencies]
 ark-ff = { version = "0.4.0", optional = true }
-blitzar-sys = { version = "0.2.0" }
+blitzar-sys = { version = "0.5.1" }
```

Comment:
Since I believe semver treats `0.2=>0.3` as a breaking change, we should probably be bumping the version more often here.

Reply:
Wonder what the best way to automate our notification to bump? Also, should we change this to `0.5` in a future PR?

Reply:
I've heard renovate is good: https://github.com/renovatebot/renovate